addons/mail/tests/test_mail_mail.py

```diff
@@ -0,0 +1,17 @@
+from odoo.tests import TransactionCase
+from unittest import mock
+import smtplib
+
+
+class MailCase(TransactionCase):
+
+    def test_mail_send_missing_not_connected(self):
+        """This assume while calling self.env['ir.mail_server'].connect() it return
+        an disconnected smtp_session which is a non falsy object value"""
```

Comment:
docstring should specify what we're testing first.
But I don't really get what your assumption is here, if you could maybe reword :)

```suggestion
        """Check getting a disconnected error while disconnecting doesn't raise.
        
        This assumes calling self.env['ir.mail_server'].connect() returns a disconnected smtp_session, which is a non-falsy object value."""
```